src/pages/LoginPage/LoginPage.tsx

```diff
@@ -2,8 +2,7 @@ import './_login_page.scss';
 import {Logo} from '../../components/Logo/Logo';
 import {LoginForm} from "../../components/LoginForm/LoginForm";
 
-type Props = {};
-export const LoginPage = (props: Props) => {
+
   return (
```

Comment:
To chyba przestanie działać po tym commicie co nie? Uciąłeś deklarację funkcji strzałkowej

Reply:
no tutaj babola odwaliłem, dobrze że jest CR :)